docs/src/Run/README.md

```diff
@@ -8,10 +8,119 @@ Look for the Quick Start guide in the Run Gateway Services section to get starte
 
 <div class="button-links">
 
-[Run Gateway Services](./GatewayServices/RunGatewayServices.md)
+[Running Gateway Services](./GatewayServices/RunGatewayServices.md)
 
 </div>
 
+## DevOps Deployment Quick Reference
+
+Refer to the following sections to get a quick overview of the minimum requirements for deploying individual Gateway Services in different environments.
+
+### Gateway Service Common Requirements
+
+The following environment variables are common to all Gateway Services. This snippet from the `docker-compose.yaml` file details the `x-common-environment` section that is required and shared across all services. Each service will have its own environment variables in addition to these common variables. Environment variables defined with the `${NAME}` syntax read their values from the shell `env`, e.g. `export NAME=VALUE`. See below for more details.
+
+```yaml
+  FREQUENCY_API_WS_URL: ${FREQUENCY_API_WS_URL:-wss://0.rpc.testnet.amplica.io}
+  SIWF_NODE_RPC_URL: ${SIWF_NODE_RPC_URL:-https://0.rpc.testnet.amplica.io}
+  REDIS_URL: 'redis://redis:6379'
+  PROVIDER_ID: ${PROVIDER_ID:-1}
+  PROVIDER_ACCOUNT_SEED_PHRASE: ${PROVIDER_ACCOUNT_SEED_PHRASE:-//Alice}
+  WEBHOOK_FAILURE_THRESHOLD: 3
+  WEBHOOK_RETRY_INTERVAL_SECONDS: 10
+  HEALTH_CHECK_MAX_RETRIES: 4
+  HEALTH_CHECK_MAX_RETRY_INTERVAL_SECONDS: 10
+  HEALTH_CHECK_SUCCESS_THRESHOLD: 10
+  CAPACITY_LIMIT: '{"type":"percentage", "value":80}'
+  SIWF_URL: 'https://projectlibertylabs.github.io/siwf/v1/ui'
+  IPFS_ENDPOINT: ${IPFS_ENDPOINT:-http://ipfs:5001}
+  IPFS_GATEWAY_URL: ${IPFS_GATEWAY_URL:-https://ipfs.io/ipfs/[CID]}
+  IPFS_BASIC_AUTH_USER: ${IPFS_BASIC_AUTH_USER:-""}
+  IPFS_BASIC_AUTH_SECRET: ${IPFS_BASIC_AUTH_SECRET:-""}
+  QUEUE_HIGH_WATER: 1000
+  CHAIN_ENVIRONMENT: 'dev'
+```
+
+Each service requires connection to a Redis instance. The `REDIS_URL` environment variable is set to `redis://redis:6379` by default. If you are using a different Redis instance, you can set the `REDIS_URL` environment variable to the appropriate connection string.
+
+Each service also requires a docker network (or equivalent) to connect to any other containers. The default network is set to `gateway_net`. If you are using a different network, you can edit the `networks:` environment variable in the `docker-compose.yaml` to the appropriate network name.
+
+Some services require a connection to an IPFS instance. See the [IPFS Setup Guide](./IPFS.md) for more information.
+
+See the [docker-compose-swarm.yaml](https://github.com/projectlibertylabs/gateway/blob/main/deployment/swarm/docker-compose-swarm.yaml) for examples of redis and ipfs services.
+
```

Comment:
This link isn't working for me.  Perhaps it isn't published yet or I don't have access.

Reply:
Yeah, that one isn't merged into main yet, it will be with this PR.